### DISCUSSION TITLE
Resolving Routes by Name with Inertia (react)

### DISCUSSION THREAD

**Discussion by lepresk** [adonisjs/core#4547] [Help]
Hello,

I recently started a project with Adonis using InertiaJS, but I'm facing a difficulty on how I can resolve routes by their name in my React components on the Inertia side, like `route('routeName', params)`. I've gone through the documentation but couldn't find anything addressing this.

Your assistance would be greatly appreciated.

**Comment by Julien-R44** [comment]
Hey, not yet

issue is tracked here : https://github.com/adonisjs/inertia/issues/14
